shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginLoader.java

```diff
@@ -184,7 +184,7 @@ public void close() {
     }
     
     @SuppressWarnings("unchecked")
-    private <T> T getOrCreateInstance(final String className) throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+    protected <T> T getOrCreateInstance(final String className) throws ClassNotFoundException, IllegalAccessException, InstantiationException {
```

Comment:
Why we use protected here? OvO

Reply:
1. mockito is not support stubbed private method in his official document. it suggest to protected. 
2. powermock is released at one year ago.

Reply:
1. It is not advisable to modify the code just for unit testing.
2. In Object Oriented, the private method is not recommended to be mocked, can we use other means instead of skipping this method?